modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/parser/hdbprocedure/XSKHDBProcedureLogger.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.hdb.ds.parser.hdbprocedure;
+
+import com.sap.xsk.utils.XSKCommonsConstants;
+import com.sap.xsk.utils.XSKCommonsUtils;
+
+public class XSKHDBProcedureLogger {
+
+  public void logError(String parsedFileLocation, String problemsFacadeMessage, String exceptionMessage) {
+    XSKCommonsUtils.logCustomErrors(parsedFileLocation,
+        XSKCommonsConstants.PARSER_ERROR,
+        "",
```

Comment:
Move these empty strings to separate constants in this class with appropriate names so that it's easier to know what they mean. Another option is to add a comment right after each `""`, your choice.